src/components/WalletStatementModal/index.tsx

```diff
@@ -51,7 +45,7 @@ function WalletStatementModal({statementPageURL, session}) {
                     title="Statements"
                     height="100%"
                     width="100%"
-                    seamless="seamless"
+                    seamless
```

Comment:
is `seamless` now a boolean prop?

Reply:
@fabioh8010 Yeah, it looks so
![image](https://github.com/Expensify/App/assets/23176449/098fe269-ef68-4509-9fd6-214fb2346b77)